client/src/Movies/Movie.js

```diff
@@ -10,10 +11,12 @@ export default class MovieCard extends React.Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the 
-    this.setState({movie:id});
+    
+    const { match : {params}} = this.props;
```

Comment:
This line tripped up many students, including myself. For more information about what is going on here, remember to study the ES6 Destructuring concept.